Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -101,14 +101,18 @@ public override void OnPreSceneQuery()
             // we need to to so using the previous touchable in TryRaisePokeUp().
             closestProximityTouchable = newClosestTouchable;
 
-            IsActive = IsNearObject;
-            visuals.SetActive(IsNearObject);
+            visuals.SetActive(IsActive);
         }
 
         public override void OnPostSceneQuery()
         {
             base.OnPostSceneQuery();
 
+            if (!IsActive)
```

Comment:
This seems interesting. Do you know if there's a reason we're running `OnPostSceneQuery` on pointers that aren't active?

Reply:
No, I'm not really sure why. It looks like the focusprovider will always update all pointer focus info regardless of it it is active or not. This pattern of return early if not active is present in other pointers, for example LinePointer.  

I'd rather not update FocusProvider right now, to avoid regressions. We can open an issue to investigate so we don't forget. Pinging @luis-valverde-ms  in case he has any thoughts.